### ISSUE TITLE
IPA diacritics. As a sticky key or combined with others?

### ISSUE THREAD

**Issue by Porrumentzio** [AnySoftKeyboard/AnySoftKeyboard#2351]
We are planning to create an International Phonetic Alphabet package for ASK. There are some issues we don't know how to solve due to technic questions.

This is the case of diacritic symbols. For example it exists in IPA a symbol, [voiceless bilabial nasal](https://en.wikipedia.org/wiki/Voiceless_bilabial_nasal), which is a combination of m [[109](https://unicode-table.com/en/006D/)] and combining ring [[805](https://unicode-table.com/en/0325/)]. 
The result is this:
![irudia](https://user-images.githubusercontent.com/61913683/85700303-86de5700-b6dc-11ea-8baf-bc6192f69c8a.png)

Here is the doubt: is it possible to enter it on some way as a **unique** symbol, 109+805, or is it mandatory to enter 805 diacritic as a **sticky key** able to function with _any_ other symbol?

**Comment by menny** [comment]
Have you seen how the NIQUD in the Hebrew pack works? It's a popup key that immediately pops-up (not as long-press)

**Comment by Porrumentzio** [comment]
Do you mean [this one](https://github.com/AnySoftKeyboard/AnySoftKeyboard/blob/master/addons/languages/hebrew/pack/src/main/res/xml/heb_qwerty_niqqud.xml)? If it's that, I don't find those pop-ups

**Comment by menny** [comment]
This is the key that pops-up the Niqqud in Hebrew: https://github.com/AnySoftKeyboard/AnySoftKeyboard/blob/master/addons/languages/hebrew/pack/src/main/res/xml/heb_qwerty_niqqud.xml#L8

You'll notice that it does not have any `codes` attribute, but has `popupLayout`. This means that when the user presses that key the popup is immediately shown. No delay.

In Hebrew, the user will type a Hebrew letter and then will pick a Niqqud from the popup. That will decorate the Hebrew letter with the picked niqqud.

Does this flow work for you?

**Comment by Luaned** [comment]
How do I write niqqud on the hebrew keyboard? Excuse me if this is not the correct place to answer this question, I am new to Github

**Comment by menny** [comment]
![Screenshot_20200824-104233](https://user-images.githubusercontent.com/481272/91058729-cf3fc580-e5f6-11ea-8818-11e204af492f.png)
![Screenshot_20200824-104302](https://user-images.githubusercontent.com/481272/91058751-d5ce3d00-e5f6-11ea-8639-7f7f1358b73c.png)

**Comment by Luaned** [comment]
Ah Ok, sorry I noticed that the lastest version on F-droid is from 2017 and that on the Google play is from this year

**Comment by Porrumentzio** [comment]
Yes, that happened also to me, and therefore I couldn't see the coherence between the code and the package, because I was downloading it from F-Droid
